x/finality/keeper/gov.go

```diff
@@ -138,7 +139,8 @@ func (k Keeper) HandleResumeFinalityProposal(ctx sdk.Context, fpPksHex []string,
 		k.SetVotingPowerDistCache(ctx, blkHeight, newDc)
 
 		// ensure every active finality provider has signing info
-		k.HandleFPStateUpdates(ctx, dc, newDc, nil)
+		// TODO: check if we should emit events at every height
```

Comment:
@GAtom22 it is emitting the events from old heights every block until the current height, we need a way to avoid calling the hooks on this